packages/vscode-extension/package.nls.json

```diff
@@ -272,13 +272,13 @@
     "teamstoolkit.localDebug.multipleTunnelServiceError": "There are multiple running local tunneling services. Close any redundant tasks to prevent conflicts.",
     "teamstoolkit.localDebug.noTunnelServiceError": "There is no running local tunneling service. Ensure that the local tunneling service has been started.",
     "teamstoolkit.localDebug.ngrokStoppedError": "Ngrok has been stopped with exit code '%s'.",
-    "teamstoolkit.localDebug.ngrokProcessError": "Failed to start ngrok. %s",
+    "teamstoolkit.localDebug.ngrokProcessError": "Failed to start ngrok.",
```

Comment:
per the latest good practice, suggest to replace all "Failed" to "Unable"

Reply:
@sffamily Updated the error messages.